packages/store-engine-sqleet/src/SchemaConverter.ts

```diff
@@ -20,20 +20,24 @@
 const uint32 = require('uint32');
 
 export const SQLeetEnginePrimaryKeyName: string = '`key`';
+export const MAGIC_SINGLE_COLUMN = '__sc_value';
```

Comment:
Let's think of a more meaningful name. How about "RESERVED_SINGLE_COLUMN"`? Is "__sc_value" the column name?